replicate/client.py

```diff
@@ -22,6 +23,22 @@ def __init__(self, api_token=None) -> None:
         # TODO: make thread safe
         self.session = requests.Session()
 
+        # Gracefully retry requests
+        # This is primarily for when iterating through predict(), where if an exception is thrown, the client
+        # has no way of restarting the iterator.
+        # We might just want to enable retry logic for iterators, but for now this is a blunt instrument to
+        # make this reliable.
+        retries = Retry(
+            total=5,
+            backoff_factor=2,
```

Comment:
Just getting a sense of what these options mean, for my own understanding.

From the [docs](https://urllib3.readthedocs.io/en/latest/reference/urllib3.util.html#module-urllib3.util.retry):

> urllib3 will sleep for:
> `{backoff factor} * (2 ** ({number of total retries} - 1))`
> seconds.

1. `2 * ( 2 ** (1-1)) = 2`
1. `2 * ( 2 ** (2-1)) = 4`
1. `2 * ( 2 ** (3-1)) = 8`
1. `2 * ( 2 ** (4-1)) = 16`
1. `2 * ( 2 ** (5-1)) = 32`